ena_upload/templates/ENA_template_experiments.xml

```diff
@@ -19,7 +29,9 @@
                         <PAIRED py:when="row.library_layout == 'paired'" NOMINAL_LENGTH="${row.insert_size}" />
```

Comment:
Just a comment for `library_layout`

In the webin submissions portal, depending on the file format chosen, the `library_layout` field is:
mandatory for: single reads fastq, paired reads fastq, 
optional for: cram, bam, Pacbio_hdf5, Oxfordnanopore_native

It seems, however, that ENA will accept a `library_layout` with value `SINGLE` for the file types in which it is optional. So it might be good to keep it as mandatory indeed.

Reply:
Since I conditional if statements are a pain, I will leave this mandatory indeed